frontend/cli/cmd_serve.go

```diff
@@ -390,7 +390,7 @@ func waitForControllerOnline(ctx context.Context, startupTimeout time.Duration,
 	ctx, cancel := context.WithTimeout(ctx, startupTimeout)
 	defer cancel()
 
-	ticker := time.NewTicker(time.Second)
+	ticker := time.NewTicker(time.Millisecond + 50)
```

Comment:
Do you want `50ms` here? Should it be `*` instead of `+`?

Reply:
oops